sidekick/lib/src/templates/cli_bundle.g.dart

```diff
@@ -3,4 +3,80 @@
\ No newline at end of file
+final sidekickBundle = MasonBundle.fromJson(<String, dynamic>{
```

Comment:
Since when is this file formatted? new mason version or have opened it accidentally triggering auto format?

Reply:
In the first attend accidentally.

Also checked VGCLI and saw that they formatted their Files too.
But with the `CI generated checker` I think I should roll back

Reply:
I'm fine with formatting it, when we put it in the generation script